homeassistant/components/forecast_solar/const.py

```diff
@@ -101,4 +101,10 @@
         device_class=DEVICE_CLASS_ENERGY,
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
     ),
+    ForecastSolarSensorEntityDescription(
```

Comment:
Can't we just use the "model" key of the device information for this, it's kinda fitting and this information is static pr. entry?

Reply:
That seems like a good plan, let's figure out how to do that.